Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/CancelDeletionProcess/HandlerTests.cs

```diff
@@ -1,18 +1,18 @@
 ﻿using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
 using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
 using Backbone.DevelopmentKit.Identity.ValueObjects;
-using Backbone.Modules.Devices.Application.Identities.Commands.CancelDeletionProcessDuringGracePeriod;
+using Backbone.Modules.Devices.Application.Identities.Commands.CancelDeletionProcess;
 using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
 using Backbone.Modules.Devices.Domain.Entities.Identities;
 using Backbone.Tooling;
 using FakeItEasy;
 using FluentAssertions;
 using Xunit;
 using static Backbone.UnitTestTools.Data.TestDataGenerator;
-using Handler = Backbone.Modules.Devices.Application.Identities.Commands.CancelDeletionProcessDuringGracePeriod.Handler;
+using Handler = Backbone.Modules.Devices.Application.Identities.Commands.CancelDeletionProcess.Handler;
```

Comment:
Here's how I wrote the import:
```csharp
using Backbone.Modules.Devices.Application.Identities.Commands.RejectDeletionProcess;
```

I think this is the usual way we do it on the project.

Reply:
My bad, `using Backbone.Modules.Devices.Application.Identities.Commands.CancelDeletionProcess;` is already on line 3. So `using Handler = Backbone.Modules.Devices.Application.Identities.Commands.CancelDeletionProcess.Handler;` is totally removed.